File: cash.c
#include <cs50.h>
#include <math.h>
#include <stdio.h>

int main(void)
{
    // dollars
    float d = 0.00;
    int count = 0;
    int c;
    //prompt a user for a number-dollar
    do
    {
        d = get_float("Change owed: ");
    }
    while (d < 0);
    c = round(d * 100); //rounding dollars and getting cents out of it
    while (c >= 25)
    {
        count++;//counting how many times the loop was executed
        c = c - 25;
    }
    while (c >= 10)
    {
        count++;
        c = c - 10;
    }
    while (c >= 5)
    {
        count++;
        c = c - 5;
    }
    while (c >= 1)
    {
        count++;
        c = c - 1;
    }
    printf("You get %d coins.\n", count);// printing how many times all the loops were executed

}

File: cash.py
from cs50 import get_float

# get the number from the user
while True:
    f = get_float("Change owed: ")
    if f > 0:
        break
# initializing count
count = 0
# turning dollars into cents and rounding the number
coins = round(f * 100)
# while there are more than 25
while coins >= 25:
    coins -= 25
    count += 1  # it's ++ in python
# while there are more than 10
while coins >= 10:
    coins -= 10
    count += 1
# while there are more than 5
while coins >= 5:
    coins -= 5
    count += 1
# while there are more than 1
while coins >= 1:
    coins -= 1
    count += 1

print("You get: ", count)